### DISCUSSION TITLE
Sort in descending order

### DISCUSSION THREAD

**Discussion by huco95** [tinacms/tinacms#3393] [Help]
Hi,

I am trying to sort some elements in descending order following these docs https://tina.io/docs/graphql/queries/advanced/pagination/ but I can manage to get it work.  How am I supposed to use the last and before arguments? Thanks!!

**Comment by erikd256** [comment]
Like this: 
-https://github.com/erikd256/stfranciscusheverlee/blob/main/.tina/queries/queries.gql
 -https://github.com/erikd256/stfranciscusheverlee/blob/main/pages/%5Bfilename%5D.tsx

**Reply by 985563349** [reply]
Thank you for your plan, but I don't quite understand why last: -1 will take effect.
